tests/TestCase/Database/Schema/PostgresSchemaDialectTest.php

```diff
@@ -620,7 +620,7 @@ public function testDescribeTableConstraintsWithKeywords(): void
             ],
         ];
         $this->assertCount(2, $result->constraints());
-        $this->assertEquals($expected['primary'], $result->getConstraint('primary'));
+        $this->assertEquals($expected['schema_authors_pkey'], $result->getConstraint('schema_authors_pkey'));
```

Comment:
This is a minor behavior change. It is arguable that the previous implementation with a static name was wrong, but it could also be interpreted as a breaking change. I don't *think* this will matter to most applications, but I'm happy to push this to 5.next if folks want.